playbooks/scenarios/test_cases/clear_clone_entry.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019 Cable Television Laboratories, Inc.
```

Comment:
DELETE ME - This is redundant with write_clone_entry.yml minus line #23 which can be sent in from the caller